### DISCUSSION TITLE
Not able to upload a good quality document as a proof for academic affiliation

### DISCUSSION THREAD

**Discussion by NicolasCarboni** [community/community#146161] [GitHub Education]
### Hi everyone,

I'm not able to do the verification of academic affiliation because I'm not able to upload a good quality document as a proof.
It is not possible to upload documents/images, and I need to take the picture from the camera of my pc, but it has a very poor quality and I'm not able to show clearly the information needed (my name, expiry date).
I tried to do the same using my mobile phone, but the problem persists. The front camera has very low quality, and when I click the option for the back camera, it doesn't change and I can only take the picture from the front camera (low quality).
I also tried to use a different browsers (chrome and Edge) and another device (tablet) but nothing change. 
What I should do to solve this?!

**Comment by quoc10973** [comment]
You try to borrow someone's phone for better quality. Take a photo that show clearly info your ID card, can include your face in the photo. Keep try again until they approve it, i think there is no another plan for this quality of photo problem
